README.md

```diff
@@ -120,48 +129,175 @@ list, I am an expert in none of these technologies).
+   [slides].
+1. Then, if you're conviced and want to create native mobile apps for Android &
+   iPhone, feel free to buy some [Full Stack courses] from Stephen Grider !
+   :tada: They aren't very expensive, 10$ each.
+1. If you're not really into JavaScript, you can try some new things like
+   [Jekyll], [Middleman] (for a portfolio) or [Hanami] (similar to Rails).
+
+#### Geeky stuff / Linux enthusiasts
+
+* You're not an Apple fan but find Ubuntu's Unity desktop a bit too meh ? Try
+   out a new Desktop Environement like [BSPWM] or [i3-gaps] and you'll discover
+   the world of Tiling window managers. You'll be more productive than ever if
+   you like the concept ! :rocket:
+
+<img src="https://i.redd.it/rh3n3dxby8fy.png" width="600">
+
+* The code editor is the most important tool for a developer. Take the time to
+   try some new stuff like [Atom] or [VScode], the other 2 popular ones besides
+   Sublime at the moment. And take the time to [custom] some nice theme, font &
+   ligatures. In pair with a [js linter], of course.
+
+<img src="https://cdn-images-1.medium.com/max/553/1*hE_nLB776KUDXPERE_3cXw.png"
+  width="400">
+
+* With the recent version of Rails (`5.1`), you should have heard the name of
+  [Webpack]. If you're a bit interested in how it works, click on the link. :)
+* You will probably not stay on Heroku forever for reasons. If you want to
+   know how to deploy your website by yourself, check this tutorial on the combo
+   [Dokku + DigitalOcean] !
+
+### Tasty treats for Web developers
+
+I personally enjoy podcasts a lot. You can listen to them anywhere (bus, car
+etc...), there a plenty of them and it's a great way to be aware of the trends
+while being busy doing something else.
+
+1. One of them is [Syntax], by Wes Bos & Scott Tolinski, creator of
+   [LevelUpTuts], a cool dude too. Their show is so funny and you learn a lot of
+   things about pretty much all.
+1. [Le Ruby Nouveau] is a more classic one but it's nice too. Oh and they have a
+   [Slack] too, join it. As my Grand Ma always says: the more Ruby Devs,
+   the better ! :muscle:
+1. [The Web Ahead], didn't listened to it yet but some people recommended it to
+   me, so it's probably nice.
 
 ## Other important resources
 
 You should know about some resources. They will help you a ton in your code
 every day. Here is a list of the main ones I can think of.
 
-* [CSS](http://cssreference.io/) and [HTML](http://htmlreference.io/) references
-* [Awesome Ruby](https://github.com/markets/awesome-ruby)
-* [Ruby](https://github.com/bbatsov/ruby-style-guide) and
-  [Rails](https://github.com/bbatsov/rails-style-guide/) style guides
-* [Ruby Weekly](http://rubyweekly.com/) newsletter
-* [Exercism](http://exercism.io/) to test your skills with great coding
-  challenges
+* [CSS] and [HTML] references
+* [Awesome Ruby]
+* A [resource] from Sylvain Peigney, similar to this topic with a ton of
```

Comment:
Please put the direct link. I cannot access it if I am not connected to slack (which I am not on the web)

Reply:
Hum ok. Actually, it is a variant of this. Agree, we can keep it

Reply:
Can we access a slack link without being connected ? I mean, it's one of my TA's article and I don't want to just copy paste out of it like that.
Moreover, since Slack is kinda private, people who are reading this article should be part of our bootcamp, so they just need to connect to Slack on their browser. Tell me if you see another way to do it properly. :)

Reply:
Yep, let's keep it like this. I don't see another way